apps/test/unit/p5lab/spritelab/SpritelabTest.js

```diff
@@ -40,11 +41,11 @@ describe('SpriteLab', () => {
     });
     afterEach(() => document.body.removeChild(container));
 
-    let studioApp;
+    let testStudioApp;
```

Comment:
naming nit: in unit tests a "fake" version of something usually called a "mock"; this object is a mock version of StudioApp, so the convention would be `mockStudioApp` (which also means your rename here is how it originally should've been named!)

Reply:
there are multiple ways to fake things in unit tests, too, as you've shown in your changes here. this is a good article about the difference between the two most common fakes, mocks and stubs: https://martinfowler.com/articles/mocksArentStubs.html